### PR TITLE
Add PostHog Detection

### PR DESCRIPTION
**Pull request by ysm-dev**
https://posthog.com/

ex)
- https://posthog.com/
- https://pry.co/
- https://www.vendasta.com/
- https://www.ycombinator.com/
- https://www.brainboard.co/

### CODE REVIEW

File: src/technologies/p.json [hunk 1]
```diff
@@ -2101,6 +2101,23 @@
...
+    "description": "PostHog is the all-in-one platform for building better products - with product analytics, feature flags, session recordings, a/b testing, heatmaps, and more.",
+    "icon": "PostHog.svg",
+    "js": {
+      "posthog": ""
+    },
+    "pricing": [
+      "freemium",
+      "recurring"
+    ],
+    "saas": true,
```
Comment (nurbek91) [inline src/technologies/p.json:2117]: `"oss":true,`

File: src/technologies/p.json [hunk 2]
```diff
@@ -2101,6 +2101,23 @@
     ],
     "website": "https://www.postaffiliatepro.com"
   },
+  "PostHog": {
+    "cats": [
+      10
+    ],
+    "description": "PostHog is the all-in-one platform for building better products - with product analytics, feature flags, session recordings, a/b testing, heatmaps, and more.",
```
Comment (nurbek91) [inline src/technologies/p.json:2108]: PostHog is the open-source, all-in-one product analytics platform.

File: src/technologies/p.json [hunk 3]
```diff
@@ -2101,6 +2101,23 @@
...
+    "icon": "PostHog.svg",
+    "js": {
+      "posthog": ""
+    },
+    "pricing": [
+      "freemium",
+      "recurring"
+    ],
```
Comment (nurbek91) [inline src/technologies/p.json:2116]:
```
 "pricing": [
      "freemium",
      "payg"
    ],
```

**Review by nurbek91 — COMMENTED** [review]
Hey mate, thanks for PR!!
A few adjustments and your PR will look perfect!

File: src/technologies/p.json (hunk 1 above)
Comment (ysm-dev) [reply src/technologies/p.json:2117]: [a59ab13](https://github.com/wappalyzer/wappalyzer/pull/7330/commits/a59ab13adcfd420ea1e95f47f142a67bd6c589a4)

File: src/technologies/p.json (hunk 3 above)
Comment (ysm-dev) [reply src/technologies/p.json]: [a59ab13](https://github.com/wappalyzer/wappalyzer/pull/7330/commits/a59ab13adcfd420ea1e95f47f142a67bd6c589a4)

**Review by nurbek91 — APPROVED** [review]

**Pull request merged**